tools/win32/load-vsenv.ps1

```diff
@@ -31,6 +31,7 @@ if (Test-Path env:BITS) {
 $vcvars_locations = @(
   "${VSBASE}\BuildTools\VC\Auxiliary\Build\vcvars${bits}.bat",
   "${VSBASE}\Community\VC\Auxiliary\Build\vcvars${bits}.bat"
+  "${VSBASE}\Enterprise\VC\Auxiliary\Build\vcvars${bits}.bat"
```

Comment:
Huh? No ,?

Reply:
[Both works](https://docs.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-arrays?view=powershell-7.2#basic-usage):

> We can declare an array on multiple lines. The comma is optional in this case and generally left out.

But it should be consistent, so I'd rather remove it as the docs sound to me like that's the preferred way.

Reply:
OK